config/local.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Remove the creation of local.json from install scripts

Reply:
Resolved as well, same comment as before.